### PR TITLE
Multichain

### PR DESCRIPTION
**Pull request by unxnn**

### CODE REVIEW

File: charts/backend/templates/common-configmap.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
 data:
   # https://github.com/notifiers/notifiers/issues/328
   USERNAME: user
-  NETWORK: {{ .Values.commonConf.network | quote }}
+  ENABLED_NETWORKS: {{ .Values.commonConf.network | quote }}
```
Comment (tsudmi) [inline charts/backend/templates/common-configmap.yaml:11]: Lets rename commonConf.network to commonConf.enabledNetworks

File: charts/backend/templates/polling-configmap.yaml [hunk 2]
```diff
@@ -8,9 +8,9 @@ metadata:
     component: "polling"
 data:
   IPFS_FETCH_ENDPOINTS: {{ .Values.polling.conf.ipfsFetchEndpoints }}
-  ETH1_ENDPOINT: {{ .Values.polling.conf.eth1Endpoint | quote }}
-  ETH2_ENDPOINT: {{ .Values.polling.conf.eth2Endpoint | quote }}
-  ETH2_ARCHIVE_ENDPOINT: {{ .Values.polling.conf.eth2ArchiveEndpoint | quote }}
+  {{ template "network_prefix" .}}_ETH1_ENDPOINT: {{ .Values.polling.conf.eth1Endpoint | quote }}
```
[the remark this answers is not in the document]
Comment (unxnn) [reply charts/backend/templates/polling-configmap.yaml]: @tsudmi added, review please

**Comment by unxnn** [comment]
> Lets expose all the variables that can be specified for backend, oracle, keeper, i.e. everything that uses config("VAR_NAME", default=...)

@tsudmi added all except hardcoded values like subgraphs urls and contract addresses. It's unlikely they will changes?

File: charts/backend/templates/polling-configmap.yaml [hunk 3]
```diff
@@ -21,4 +33,5 @@ data:
   ACTIVATED_DEPOSITS_NOTIFY_SECONDS: {{ .Values.polling.conf.activatedDepositsNotifySeconds | quote }}
   POLL_VALIDATORS_PERIOD: {{ .Values.polling.conf.pollValidatorsPeriod | quote }}
   POLL_UNISWAP_PERIOD: {{ .Values.polling.conf.pollUniswapPeriod | quote }}
+  POLL_STAKERS_REWARDS: {{ .Values.polling.conf.pollStakersRewards | quote }}
```
Comment (tsudmi) [inline charts/backend/templates/polling-configmap.yaml:36]:
```suggestion
  POLL_STAKERS_REWARDS_PERIOD: {{ .Values.polling.conf.pollStakersRewardsPeriod | quote }}
```

Comment (tsudmi) [inline charts/backend/templates/polling-configmap.yaml:36]: POLL_VALIDATORS_SNAPSHOTS_PERIOD, POLL_VALIDATOR_BILLS_PERIOD are missing

File: charts/backend/templates/polling-configmap.yaml [hunk 4]
```diff
@@ -21,4 +33,5 @@ data:
   ACTIVATED_DEPOSITS_NOTIFY_SECONDS: {{ .Values.polling.conf.activatedDepositsNotifySeconds | quote }}
```
Comment (tsudmi) [inline charts/backend/templates/polling-configmap.yaml:33]: REALITY_ETH_QUESTION_URL_TEMPLATE, VOTING_PAGE, ACTIVATED_DEPOSITS_NOTIFY_SECONDS were moved to base_settings, they must be in common configmap

File: charts/keeper/values.yaml [hunk 5]
```diff
@@ -39,14 +39,16 @@ priorityClassName: ""
 
 settings:
   logLevel: "INFO"
-  network: "mainnet"
-  web3Endpoint: ""
+  enabledNetworks: "mainnet"
+  eth1Endpoint: ""
```
Comment (tsudmi) [inline charts/keeper/values.yaml:43]: It should be possible to specify eth1Endpoint for multiple networks

File: charts/keeper/values.yaml [hunk 6]
```diff
@@ -39,14 +39,16 @@ priorityClassName: ""
...
   healthServerHost: "0.0.0.0"
   healthServerPort: "8080"
   confirmationBlocks: "15"
   keeperProcessInterval: "10"
   transactionTimeout: "900"
+  minBalanceWei: "100000000000000000"
+  maxFeePerGasGwei: "150"
 
   # credentials
   oraclePrivateKey: ""
```
Comment (tsudmi) [inline charts/keeper/values.yaml:54]: It should be possible to specify oraclePrivateKey for multiple networks

**Comment by tsudmi** [comment]
@unxnn please review

**Comment by unxnn** [comment]
> @unxnn please review

@tsudmi Everything is ok, let's merge

**Review by tsudmi — APPROVED** [review]

**Pull request merged**